open-source.php

```diff
@@ -155,6 +155,38 @@
     </div>
 </div>
 
+<div class="grid">
+    <div class="whole">
+        <div class="two-thirds">
+            <h2>Projects We've Supported</h2>
```

Comment:
`Projects We Support` sounds better I think, as some of these are recurring and not just one-off "yeah we supported them one time".